dashboard/test/controllers/followers_controller_test.rb

```diff
@@ -94,6 +94,21 @@ class FollowersControllerTest < ActionController::TestCase
     assert_equal @chris_section, follower.section
   end
 
+  test "student_user_new errors when joining a section with deleted teacher" do
+    @laurel.update(deleted_at: Time.now)
```

Comment:
Not to get too off the rails here in this PR, but her birthday is tomorrow, so it's especially on my mind